src/getHumanAge.test.js

```diff
@@ -7,4 +7,21 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [1, 1] if animalAge < first + second', () => {
+    expect(getHumanAge(20, 20)).toEqual([1, 1]);
+  });
+
+  test('should return [0, 0] if animalAge === 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [2, 2] if animalAge > first + second', () => {
+    expect(getHumanAge(26, 26)).toEqual([2, 2]);
+  });
+
+  test('should return [3, 3] if more than'
+  + 'first and second and after divide by other', () => {
+    expect(getHumanAge(29, 29)).toEqual([3, 3]);
```

Comment:
The test description is unclear. It mentions 'more than first and second and after divide by other', but it's not clear what 'other' refers to. Ensure the description accurately reflects the logic being tested.